src/ParallelAlgorithms/Tags.hpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Utilities/PrettyType.hpp"
+
+namespace Parallel::Tags {
```

Comment:
I think it would be better to create a `Tags` subdir and then have `IterationId.hpp` in there. The reason being that I can see this `Tags.hpp` collecting a lot of unrelated tags. Does that seem reasonable?

Reply:
Sounds good. Do you have a better idea for the namespace?

Reply:
If you mean better than `Tags`, no I have not come up with anything else :P

Reply:
:D no sorry, I mean an alternative to `Parallel`. I was wondering if we should distinguish parallel algorithms from the parallel backend.

Reply:
Ah, okay. I agree we should distinguish. Maybe `Parallel::Algorithms`? I don't have anything better at the moment

Reply:
That's so long.. How about `palgs`? Has a nice quirky ring to it :D

Reply:
(just for humor: parallalgs)

Reply:
I like the sound of that, I don't like typing it :D

Reply:
I could also move these tags to `NumericalAlgorithms/Convergence/` and put them in the `Convergence` namespace. Do you like that better?